modules/admin-ui/src/main/resources/public/org/opencastproject/adminui/languages/lang-en_US.json

```diff
@@ -872,15 +872,18 @@
           "WORKFLOWS": {
             "ACTIONS": "Actions",
             "DETAILS": "Details",
-            "DESCRIPTION": "",
-            "MORE_INFO": "More Information & Actions",
-            "ID": "ID",
+            "DESCRIPTION": "Description",
+            "MORE_INFO": "More Information",
+            "ID": "WIID",
             "TYPE": "Type",
             "TITLE": "Title",
-            "STATE": "State",
-            "STARTED": "Started",
+            "STATUS": "Status",
+            "STARTED": "Started At",
             "SERIES": "Series",
-            "SUBMITTED": "Submitted",
+            "SUBMITTED_AT": "Submitted At",
+            "SUBMITTED_BY": "Submitted By",
```

Comment:
`Suggestion: `Submitter`

Reply:
I prefer "Submitted by". Very symmetric to "Submitted at". Reads easy and looks good (see screenshots).

Reply:
Consistency argument:

- It's “description” not “described as”
- It's “presenter” not “presented by”
- it's “contributor” not “contributed by”
- It's “start date” in the metadata tab, not “started at”
- …

Reply:
Note that in the case of workflows, for example, we have two levels of metadata: Metadata describing the actual object itself (e.g. title, description, ...) and data about who has created the object at what point of time (submitted by, submitted at).

Currently, the "... by" and "... at" are used at several locations in the UI when it comes to the meta-metadata, e.g. Event: "Erstellt am"/but admittedly "Created", Series: "Erstellt von"/"Created By".

Personally - but that is a matter of taste - I would prefer going for the "... by" and "... at" variants across the UI instead of replacing the "... by" by "..." and "... at" by "... date".

Mostly because "Submitted by"/"Übermittelt von" seems more friendly to me as "Submitter"/"Übermittler". Same for "Submitted at"/"Übermittelt am" vs "Submission date"/"Übermittlungsdatum".

It might even help to distinguish between metadata vs meta-metadata.

Anyway, will bring up the point on the Today's technical meeting to see what others think.

Reply:
I don't think your "meta-metadata" declaration makes sense. Both are metadata about the job…

But in any case, I'm generally fine with changing these keys/labels. I don't really care for one variant or the other. But then let's do that consistently across the UI.

And please also consider that doing that has the downside of maybe invalidating a somewhat large (no idea how many keys we would need to touch) portion of our translation for arguably relatively little gain.

Again, we can do that. I have no intention on stopping that. But modifying just these keys looks weird.

Reply:
Changed to "Submitter" and "Submitted"